mmv1/third_party/tgc/tfdata/fake_resource_data.go

```diff
@@ -92,6 +98,10 @@ func (d *FakeResourceData) Get(name string) interface{} {
 	return val
 }
 
+func (d *FakeResourceData) GetRawConfig() cty.Value {
+	return d.GetRawConfig()
```

Comment:
Does the function GetRawConfig call itself? Or I miss something.

Reply:
i've moved it to the list of mappers that are required but not needed or used in tgc